game-service/game-infra/src/main/kotlin/com/grepp/quizy/infra/game/websocket/CustomHandshakeHandler.kt

```diff
@@ -0,0 +1,19 @@
+package com.grepp.quizy.infra.game.websocket
+
+import org.springframework.http.server.ServerHttpRequest
+import org.springframework.web.socket.WebSocketHandler
+import org.springframework.web.socket.server.support.DefaultHandshakeHandler
+import java.security.Principal
+import java.util.*
+
+class CustomHandshakeHandler : DefaultHandshakeHandler() {
+
+    override fun determineUser(
+        request: ServerHttpRequest,
+        wsHandler: WebSocketHandler,
+        attributes: MutableMap<String, Any>
+    ): Principal {
```

Comment:
https://kotlinlang.org/docs/coding-conventions.html#functions

Kotlin 공식 코딩 컨벤션 찾다가 발견한 부분인데
단일 return 문으로 구성된 함수는 따로 블럭으로 두지 않고
그냥 `=`으로 작성하는 걸 지향한다고 합니다!

```suggestion
    ): Principal = WebSocketPrincipal(UUID.randomUUID().toString())
```

반환 타입도 링크 예시처럼 안 봐도 바로 알겠는 것들은 생략하기도 하는 거 같더라구요

Reply:
오... 이런게 있었군요 최대한 신경써보도록 하겠습니다... 처음하다보니 컨벤션 최대한 맞추면서 하려고하는데 어렵네요